src/main/java/net/sf/jabref/groups/GroupSelector.java

```diff
@@ -74,6 +74,8 @@
 import net.sf.jabref.logic.search.rules.sets.SearchRuleSets;
 import net.sf.jabref.logic.search.rules.sets.SearchRuleSet;
 import net.sf.jabref.gui.undo.NamedCompound;
+import net.sf.jabref.gui.util.GUIGlobals;
+
```

Comment:
Yes, I created that package to contain gui-related "utility" classes and methods (i.e., small useful stuff...).